include/rocksdb/statistics.h

```diff
@@ -396,7 +396,7 @@ enum StatsLevel {
   // Collect all stats except the counters requiring to get time inside the
   // mutex lock.
   kExceptTimeForMutex,
-  // Collect all stats expect time inside mutex lock AND time spent on
+  // Collect all stats except time inside mutex lock AND time spent on
   // compression
   kExceptDetailedTimers,
```

Comment:
What if we moved `kExceptDetailedTimers` before `kExceptTimeForMutex`
We can do the check in format.cc to be `stats_level_ > kExceptDetailedTimers`
and in instrumented_mutex.cc to be `stats_level_ > kExceptTimeForMutex`